Mathlib/LinearAlgebra/TensorProduct/Basis.lean

```diff
@@ -53,6 +53,10 @@ theorem tensorProduct_repr_tmul_apply (b : Basis ι S M) (c : Basis κ R N) (m :
     (tensorProduct b c).repr (m ⊗ₜ n) (i, j) = c.repr n j • b.repr m i := by
   simp [tensorProduct]
 
+lemma coe_tensorProduct (b : Basis ι R M) (c : Basis κ R N) :
```

Comment:
I'd be tempted to drop this lemma; you don't need it, and can use `tensorProduct_apply'` in your theorem below instead. @erdOne, do you have opinions?